src/libYARP_dev/include/yarp/dev/RGBDSensorParamParser.h

```diff
@@ -66,16 +66,28 @@ class YARP_dev_API RGBDSensorParamParser
         YARP_SUPPRESS_DLL_INTERFACE_WARNING_ARG(std::vector<yarp::os::Value>) val;
     };
 
-    IntrinsicParams         depthIntrinsic;
-    IntrinsicParams         rgbIntrinsic;
-    yarp::sig::Matrix       transformationMatrix;
-    bool                    isOptionalExtrinsic;
 
+    /**
+     * @brief RGBDSensorParamParser, default constructor.
+     */
     RGBDSensorParamParser(): depthIntrinsic(), rgbIntrinsic(),
                              transformationMatrix(4,4), isOptionalExtrinsic(true) {
         transformationMatrix.eye();
     }
-    bool parseParam(yarp::os::Searchable& config, std::vector<RGBDParam *> &params);
+
+    /**
+     * @brief parseParam, parse the params stored in a Searchable.
+     * @param config[in], Searchable containing the parameters of the RGDB sensor
+     * @param params[out], vector containing all the description/settings of the
+     * RGBD sensor.
+     * @return true on success, false otherwise.
+     */
+    bool parseParam(const yarp::os::Searchable& config, std::vector<RGBDParam *> &params);
+
+    yarp::sig::IntrinsicParams         depthIntrinsic;
```

Comment:
Small comment for this and the following three lines: what with this indentation 😄 ?